npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -361,6 +390,10 @@ func (iMgr *IPSetManager) RemoveFromList(listMetadata *IPSetMetadata, setMetadat
 	modified := false
 	for _, setMetadata := range setMetadatas {
 		memberName := setMetadata.GetPrefixName()
+		if memberName == "" {
+			metrics.SendErrorLogAndMetric(util.IpsmID, "[RemoveFromList] warning: removing invalid member from list %s, member %s", list.Name, memberName)
```

Comment:
unnecessary check because of !exists --> continue below

Reply:
What if due a bug a set with name "" got created? this check will safeguard such issue.